libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/PredictiveCache.kt

```diff
@@ -48,6 +51,7 @@ object PredictiveCache {
     }
 
     fun createMapsController(
+        mapboxMap: Any,
         tileStore: TileStore,
         tileVariant: String,
         predictiveCacheLocationOptions: PredictiveCacheLocationOptions =
```

Comment:
Noticed that we're not passing `predictiveCacheLocationOptions` in any of the call-sites 👀 

https://github.com/mapbox/mapbox-navigation-android/blob/1e4b0ea1033dbc3c6986f4052172dc3cf15f4992/libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt#L86

Should we? Wondering if now that we're going to allow multiple instances if these can be configurable when it comes to maps `PredictiveCacheLocationOptions`. If not, we can probably remove this parameter and create the `PredictiveCacheLocationOptions` internally (instead of by default).

Reply:
have thought about it, let's move it to 
```kotlin
fun createMapControllers(
        map: MapboxMap,
        cacheLocationOptions: PredictiveCacheLocationOptions =
            PredictiveCacheLocationOptions.Builder().build(),
        sourceIdsToCache: List<String> = emptyList()
    )
``` 
to make it flexible